packages/ui/src/components/Typography/Typography.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactNode, ReactText } from 'react';
+import { Box, forwardRef, HTMLChakraProps } from '@chakra-ui/react';
+
+type TypographyVariant = 'span' | 'div' | 'a' | 'p' | 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+
+type TypographyFontSize = 'xs' | 'sm' | 'md' | 'lg' | 'xl' | '2xl' | '3xl' | '4xl' | '5xl' | '6xl' | '7xl';
+
+type TypographyFontWeight = 'regular' | 'medium';
+
+export interface BoxProps extends HTMLChakraProps<'div'> {
```

Comment:
It's a Typography component so it would be better to name this interface `TypographyProps`